src/services/external/accounts.filing.service.ts

```diff
@@ -16,31 +18,106 @@ export class AccountsFilingService {
      * @returns {Promise<Resource<AccountValidatorResponse>>} A promise resolving to the account validation response.
      * @throws {Error} Throws an error if the response status is not 200 or if no resource is included in the response.
      */
-    async getValidationStatus(req: AccountsFilingValidationRequest): Promise<Resource<AccountValidatorResponse>> {
+    async getValidationStatus(
+        req: AccountsFilingValidationRequest
+    ): Promise<Resource<AccountValidatorResponse>> {
         const fileId = req.fileId;
         logger.debug(`Getting validation status for file ${fileId}`);
 
-        const accountsFilingService = this.privateApiClient.accountsFilingService;
-        const accountValidatorResponse = await accountsFilingService.checkAccountsFileValidationStatus(req);
+        const accountsFilingService =
+            this.privateApiClient.accountsFilingService;
+        const accountValidatorResponse =
+            await accountsFilingService.checkAccountsFileValidationStatus(req);
 
-        logger.debug(`Response for ${fileId}: ${JSON.stringify(accountValidatorResponse, null, 2)}`);
+        logger.debug(
```

Comment:
I'm not a big fan of expanding these log lines onto a ton of lines, makes the function a lot longer.